autobotAI_integrations/integrations/nessus/__init__.py

```
@@ -61,6 +64,113 @@ def get_scan_details(self, scan_id: int) -> requests.Response:
+    def stop_scan(self, scan_id: int) -> requests.Response:
+        """Stop a running vulnerability scan."""
+        endpoint = f"{self.url}/scans/{scan_id}/stop"
+        return requests.post(endpoint, headers=self._headers(), verify=self.verify_ssl, timeout=30)
+
+    def kill_scan(self, scan_id: int) -> requests.Response:
+        """Forcefully terminate a running vulnerability scan faster than stop."""
+        endpoint = f"{self.url}/scans/{scan_id}/kill"
+        return requests.post(endpoint, headers=self._headers(), verify=self.verify_ssl, timeout=30)
+
+    def export_scan(
+        self, scan_id: int, format: str = "nessus", chapters: Optional[str] = None
```

Ruff (0.15.20)

[error] Function argument `format` is shadowing a Python builtin

(A002)